code/modules/clothing/suits/ego_gear/aleph.dm

```diff
@@ -73,3 +73,16 @@
 							TEMPERANCE_ATTRIBUTE = 120,
 							JUSTICE_ATTRIBUTE = 120
 							)
+
+/obj/item/clothing/suit/armor/ego_gear/adoration
+	name = "adoration"
+	desc = "It is not as unpleasant to wear as it is to look at. \
+	In fact, it seems to give you an illusion of comfort and bravery."
+	icon_state = "adoration"
+	armor = list(RED_DAMAGE = 80, WHITE_DAMAGE = 50, BLACK_DAMAGE = 80, PALE_DAMAGE = 50)
```

Comment:
Wait, why does it have 80% in red and black?
In LC it was 0.3/0.6/0.3/1, so, 70/40/70/50 there.